crates/radix-engine-toolkit/src/transaction_types/traverser/types/general.rs

```diff
@@ -144,7 +144,8 @@ impl GeneralDetector {
                         | EntityType::GlobalAccessController
                         | EntityType::GlobalOneResourcePool
                         | EntityType::GlobalTwoResourcePool
-                        | EntityType::GlobalMultiResourcePool => FnRules::all_disallowed(),
+                        | EntityType::GlobalMultiResourcePool
+                        | EntityType::GlobalAccountLocker => FnRules::all_disallowed(),
```

Comment:
Based on the discussion in Slack this should be in `FnRules::all_allowed()`. Can we please change that and also add a test?